content/ruby/concepts/comments/comments.md

```diff
@@ -32,3 +32,21 @@ I am a multi line comment.
 I can take as many lines as needed.
 =end
 ```
+
+## Example
+
+The code below uses single-line comments.
+
+```rb
+# This example uses a loop to count only the even values
+count = 0
+
+for i in 1..10 do
+  if (i % 2 == 0)
+    count += i
```

Comment:
given the comment above, I think this should be count += 1. Or line 41 can be changed to explain that the example will sum up the even values. I think it'd also be helpful to add some additional comments like when setting the variable `count` and then also looping through a range, etc